src/state.rs

```diff
@@ -16,14 +16,13 @@ use crate::vote::{DkgSignedVote, DkgVote, IdAck, IdPart, NodeId};
 
 /// State of the Dkg session, contains the sync keygen and currently known Parts and Acks
 /// Can handle votes coming from other participants
-pub struct DkgState<R: bls::rand::RngCore> {
+pub struct DkgState {
```

Comment:
Can we derive `Debug` for `DkgState` now?

Reply:
`DkgState` contains `SecretKey` that's why we can't derive `Debug`